crypto/client.go

```diff
@@ -15,8 +16,16 @@ func New(config Config) (Client, error) {
 		if config.Type != "" {
 			fmt.Println("Invalid encryption type, using default")
 		}
-		return NewAESOpenPGPClient(config)
-	case "aes-openpgp":
-		return NewAESOpenPGPClient(config)
+		fallthrough
+	case "aes_openpgp":
+		// Remove other settings
+		// TODO(leon): This is shitty.
+		config.Settings.AESGCMSettings = nil
```

Comment:
I don't like this approach. We shouldn't check the crypto type in initConfig. This would only move the problem to a different function.
It's fine to nil it out in crypto.New, I just don't like the way it is done: There should be a 'cryptoConfigNilOutEverthingExcept(type string)' function.